src/bindgen/ir/generics.rs

```diff
@@ -51,3 +54,87 @@ impl Source for GenericParams {
         }
     }
 }
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash, PartialOrd, Ord)]
+pub struct Generic {
```

Comment:
Is there a reason you renamed this from `GenericPath` to just `Generic`? I think I prefer the old name and keeping it in `path.rs`, but it's not super important.